src/os/ogc/wfs/wfsformatter.js

```diff
@@ -8,11 +8,13 @@ goog.require('os.ogc');
 
 /**
  * Create a WFS request payload.
- *
+ * @param {Object<string, string>=} opt_params params to override uri format
  * @implements {os.net.IDataFormatter}
  * @constructor
  */
-os.ogc.wfs.WFSFormatter = function() {};
+os.ogc.wfs.WFSFormatter = function(opt_params) {
+  this.params = opt_params ? opt_params : {};
```

Comment:
Adding param overrides seems like unnecessary bloat to this class.

The `format` call already reads params from `uri.getQueryData()`. If you need to modify an existing `goog.Uri` instance, use `otherUri.clone()` and modify the clone before passing to `format`.